applications/dotnix.md

```diff
@@ -0,0 +1,240 @@
+# Dotnix
+
+- **Team Name:** Sporyon Technologies OÜ
+
+- **Payment Address:** In the case of fiat payment, please share your bank account privately with grants@web3.foundation  Fiat 25.03.2024, 15:18
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview 📄
+
+If this application is in response to an RFP, please indicate this on the first line of this section.
+
+If this is an application for a follow-up grant (the continuation of an earlier, successful W3F grant), please provide name and/or pull request of said grant on the first line of this section.
```

Comment:
```suggestion
```

Reply:
This is neither a response to an RFP nor an application a follow-up grant.